suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -784,3 +785,19 @@ export const hasNetworkFeatures = (
 
     return areFeaturesPresent;
 };
+
+// AccountUtxo
+export const getUtxoOutpoint = (utxo: { txid: string; vout: number }) => {
```

Comment:
I would maybe put this into `pacages/crypto-utils` package since it might be useful in:
- connect
- future metadata/labeling package

but no need to do it now, this is easy to move it around anytime later

Reply:
yeah i was thinking about moving it into crypto-utils, but it is not used either in connect or utxo-lib yet